libs/botan/Makefile

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (C) 2019 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=botan
+PKG_VERSION:=2.10.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=BangLang Huang <banglang.huang@foxmail.com>
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_URL:=https://codeload.github.com/randombit/botan/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=b5f27e65e733fb43c0802aef8f86ff981eb47bcfeafbf085233a3e4046e3cba8
+
+PKG_CAT:=zcat
```

Comment:
Needed at all?

Reply:
removed

Reply:
```
touch /home/build/build_dir/build_dir/target-mips_24kc_musl/botan-2.10.0/.prepared_f1b12edfae2c413ec155073efda5ffb3_18f1e190c5d53547fed41a3eaa76e9e9_check
. /home/build/build_dir/include/shell.sh;  /home/build/build_dir/dl/botan-2.10.0 | tar -C /home/build/build_dir/build_dir/target-mips_24kc_musl/botan-2.10.0/.. -xf - 
bash: /home/build/build_dir/dl/botan-2.10.0: Permission denied
/home/build/build_dir/staging_dir/host/bin/tar: This does not look like a tar archive
/home/build/build_dir/staging_dir/host/bin/tar: Exiting with failure status due to previous errors
```

It seems the download method can not recognize the format correctly.